buildrunner/__init__.py

```diff
@@ -365,10 +365,31 @@ def run(self):  # pylint: disable=too-many-statements,too-many-branches,too-many
                     step_name,
                     step_config,
                 ) in buildrunner_config.run_config.steps.items():
+                    # Reset the cache_from and cache_to for each step to the global values
+                    multi_platform._cache_from = (
```

Comment:
Accessing private variables is an anti-pattern, maybe you can use a method instead?

Reply:
Added set methods